cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,491 @@ Cask Data Application Platform Release Notes
+  
+- `CDAP-3693 <https://issues.cask.co/browse/CDAP-3693>`__ -
+  Added support for CDAP to work with HDP 2.3.
+
+
+Improvements
+------------
+
+- `CDAP-1914 <https://issues.cask.co/browse/CDAP-1914>`__ -
+  Added documentation of the RESTful endpoint to retrieve the properties of a stream.
+
+- `CDAP-2514 <https://issues.cask.co/browse/CDAP-2514>`__ -
+  Added an interface to load a file into a stream from the CDAP-UI.
+  
+- `CDAP-2809 <https://issues.cask.co/browse/CDAP-2809>`__ -
+  The CDAP-UI "Errors" pop-up in the main screen now displays the time and date for each
+  error.
+    
+- `CDAP-2872 <https://issues.cask.co/browse/CDAP-2872>`__ -
+  Updated the Cloudera Manager CSD to use support for logback.
+  
+- `CDAP-2950 <https://issues.cask.co/browse/CDAP-2950>`__ -
+  Cleaned up the messages shown in the errors dropdown in the CDAP-UI.
+
+- `CDAP-3147 <https://issues.cask.co/browse/CDAP-3147>`__ -
+  Added a  CDAP-CLI command to stop a workflow.
+  
+- `CDAP-3179 <https://issues.cask.co/browse/CDAP-3179>`__ -
+  Added support for upgrading the Hadoop distribution or the HBase version that CDAP is
+  running on.
+  
+- `CDAP-3257 <https://issues.cask.co/browse/CDAP-3257>`__ -
+  Revised the documentation of the file ``cdap-default.xml``, removed properties no longer
+  in use, and corrected discrepancies between the documentation and the shipped XML
+  file.
+
+- `CDAP-3270 <https://issues.cask.co/browse/CDAP-3270>`__ -
+  Improved the help provided in the CDAP-CLI for the setting of stream formats.
+  
+- `CDAP-3275 <https://issues.cask.co/browse/CDAP-3275>`__ -
+  Upgraded netty-http version to 0.12.0.
+  
+- `CDAP-3282 <https://issues.cask.co/browse/CDAP-3282>`__ -
+  Added a HTTP RESTful API to update the application configuration and artifact version.
+  
+- `CDAP-3332 <https://issues.cask.co/browse/CDAP-3332>`__ -
+  Added a "clear" button in the CDAP-UI for cases where a user decides to not used a
+  pre-populated schema.
+  
+- `CDAP-3351 <https://issues.cask.co/browse/CDAP-3351>`__ -
+  Defined a directory structure to be used for predefined applications.
+  
+- `CDAP-3357 <https://issues.cask.co/browse/CDAP-3357>`__ -
+  Added documentation in the source code on adding new commands and completers to the CDAP-CLI.
+
+- `CDAP-3393 <https://issues.cask.co/browse/CDAP-3393>`__ -
+  In the CDAP-UI, added visualization for Workflow tokens in Workflows.
+  
+- `CDAP-3443 <https://issues.cask.co/browse/CDAP-3443>`__ -
+  Added an example cdap-env.sh to the shipped packages.
+  
+- `CDAP-3464 <https://issues.cask.co/browse/CDAP-3464>`__ -
+  Added an example in the documentation explaining how to prune invalid transactions from
+  the transaction manager.
+
+- `CDAP-3490 <https://issues.cask.co/browse/CDAP-3490>`__ -
+  Modified the CDAP upgrade tool to delete all adapters and the ETLBatch and ETLRealtime
+  ApplicationTemplates.
+  
+- `CDAP-3493 <https://issues.cask.co/browse/CDAP-3493>`__, 
```

Comment:
Missing description? Actually the resolution of one of these is duplicate, can we remove that?